GPU/GPUTracking/Base/GPUSettingsList.h

```diff
@@ -43,6 +43,11 @@ AddOption(MaxTrackQPt, float, 1.f / GPUCA_MIN_TRACK_PT_DEFAULT, "", 0, "required
 AddOption(SearchWindowDZDR, float, 2.5, "", 0, "Use DZDR window for seeding instead of vertex window")
 AddOption(TrackReferenceX, float, 1000.f, "", 0, "Transport all tracks to this X after tracking (disabled if > 500, auto = 1000)")
 AddOption(tpcZSthreshold, float, 2.0f, "", 0, "Zero-Suppression threshold")
+AddOption(tpcCFqmaxCutoff, float, 3.0f, "", 0, "Cluster Finder rejects cluster with qmax below this threshold")
+AddOption(tpcCFqtotCutoff, float, 0.f, "", 0, "Cluster Finder rejects cluster with qtot below this threshold")
+AddOption(tpcCFinnerThreshold, float, 0.f, "", 0, "Cluster Finder extends cluster if inner charge above this threshold")
+AddOption(tpcCFminSplitNum, float, 1.f, "", 0, "Minimum number of split charges in a cluster for the cluster to be marked as split")
+AddOption(tpcCFnoiseSuppressionEpsilon, float, 10.f, "", 0, "Cluster Finder: Difference between peak and charge for the charge to count as a minima during noise suppression")
```

Comment:
We can save some constant space here if we use unsigned char instead of float.

Reply:
Fixed.